heartbeat/monitors/active/http/_meta/fields.yml

```diff
@@ -13,6 +13,16 @@
           migration: true
           description: >
             Service url used by monitor.
+        - name: response
+          type: group
+          fields:
+            - name: body
+              type: group
+              fields:
+                - name: hash
+                  type: keyword
+                  description: >
+                    Hash of the response body. Can be used to group responses with identical hashes
```

Comment:
Is this the hash of the full response body of the partial response body that is returned?

Reply:
The full body, clarified it in the latest push.